src/main/scala/com/amazon/deequ/checks/Check.scala

```diff
@@ -130,6 +131,36 @@ case class Check(
     addFilterableConstraint { filter => completenessConstraint(column, assertion, filter, hint) }
   }
 
+  /**
+    * Creates a constraint that asserts on completion in combined set of columns.
+    *
+    * @param columns Columns to run the assertion on
+    * @param hint A hint to provide additional context why a constraint could have failed
+    * @return
+    */
+  def isCombinedComplete(
+      columns: Seq[String],
+      hint: Option[String] = None)
+    : CheckWithLastConstraintFilterable = {
+    satisfies(isEachNonNull(columns), "Combined Completeness", Check.IsOne, hint)
+  }
+
+  /**
+    * Creates a constraint that assert on completion in combined set of columns.
+    *
+    * @param columns Columns to run the assertion on
+    * @param assertion Function that receives a double input parameter and returns a boolean
+    * @param hint A hint to provide additional context why a constraint could have failed
+    * @return
+    */
+  def hasCombinedCompleteness(
```

Comment:
rename this to 'haveCombinedCompleteness'

Reply:
should we include `Combined` in the name? maybe `haveCompleteness` to follow the same convention we used in `areComplete`?

Reply:
Yes, good point, lets call it haveCompleteness

Reply:
done